src/requirements/reqs.json

```diff
@@ -544,514 +560,6 @@
-                            "search": "code",
-                            "includes": [
-                                [
-                                    "AEM 3245",
-                                    "AEM 3249",
-                                    "AEM 3270",
-                                    "AEM 3450",
-                                    "AEM 4016",
-                                    "AEM 4080",
-                                    "AEM 4096",
-                                    "AEM 4150",
-                                    "AEM 4160",
-                                    "AEM 4400",
-                                    "AEM 4420",
-                                    "AEM 4435",
-                                    "AEM 4495",
-                                    "AEM 4550"
-                                ]
-                            ],
-                            "fulfilledBy": "credits",
-                            "minCount": 9,
-                            "majorDep": true
-                        }
-                    ]
-                },
-                {
-                    "name": "Strategy",
-                    "requirements": [{
-                            "name": "Part 1",
-                            "search": "code",
-                            "includes": [
-                                [
-                                    "AEM 3220",
-                                    "AEM 4160",
-                                    "AEM 4190",
-                                    "AEM 4371",
-                                    "AEM 4380",
-                                    "AEM 4390",
-                                    "AEM 4615"
-                                ]
-                            ],
-                            "fulfilledBy": "credits",
-                            "minCount": 6,
-                            "majorDep": true
-                        },
-                        {
-                            "name": "Part 2",
-                            "search": "code",
-                            "includes": [
-                                [
-                                    "AEM 2550",
-                                    "AEM 3110",
-                                    "AEM 3245",
-                                    "AEM 3260",
-                                    "AEM 3249",
-                                    "AEM 3320",
-                                    "AEM 4010",
-                                    "AEM 4015",
-                                    "AEM 4080",
-                                    "AEM 4580",
-                                    "AEM 4421"
-                                ]
-                            ],
-                            "fulfilledBy": "credits",
-                            "minCount": 9,
-                            "majorDep": true
-                        }
-                    ]
-                }
```

Comment:
What's happening here? Why so many lines are deleted?

Reply:
During the Planathon, Kathleen added a bunch of useful data for AEM concentration requirements. However, the current platform doesn't support concentrations within majors. We removed the data temporarily until our code is mature enough to support it. Have it saved and gonna get back to this soon enough